src/components/ConvictionBanner.js

```diff
@@ -0,0 +1,123 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { useTransition } from 'react-spring'
+import banner from '../base/banner'
+import {
+  ButtonBase,
+  IconCross,
+  springs,
+  GU,
+  RADIUS,
+  useViewport,
+} from '@1hive/1hive-ui'
+const BANNER_HEIGHT = 48
+const MOBILE_BANNER_HEIGHT = 64
+
+function ConvictionBanner() {
+  const [visible, setVisible] = useState(!!banner.text)
+  const { below } = useViewport()
+
+  const compactMode = below('medium')
+  useEffect(() => {
+    let timeout
+    if (compactMode && visible) {
+      timeout = setTimeout(() => setVisible(false), 5000)
+    }
+
+    return () => clearTimeout(timeout)
+  }, [compactMode, visible])
+
+  const toggleVisible = useCallback(() => {
+    setVisible(v => !v)
+  }, [setVisible])
+
+  const transitions = useTransition(visible, null, {
+    from: { height: 0 },
+    enter: { height: compactMode ? MOBILE_BANNER_HEIGHT : BANNER_HEIGHT },
+    leave: { height: 0 },
+    config: springs.smooth,
+  })
+
+  return transitions.map(({ item: visible, key, props }) => {
+    return (
+      visible && (
+        <div
+          key={key}
+          style={{ ...props, overflow: 'hidden' }}
+          css={`
+            width: 100%;
+            background: linear-gradient(
```

Comment:
maybe use the colors from [here](https://github.com/TECommons/tec-branding-assets)